engines/scc_proxy/lib/scc_proxy/engine.rb

```diff
@@ -318,6 +352,12 @@ def scc_activate_product
               error = JSON.parse(response.body)
               logger.info "Could not activate #{@product.product_string}, error: #{error['error']} #{response.code}"
               error['error'] = SccProxy.parse_error(error['error']) if error['error'].include? 'json'
+              if @system.payg?
+                # if trying to activate first product on a hybrid system
+                # it means the system was "just" announced on this call
+                # if product activation failed, system should get de-register from SCC
+                de_register_hybrid(request.headers['HTTP_AUTHORIZATION'])
```

Comment:
s/de_register/deregister/

Reply:
I do not quite understand why this was resolved. Everywhere else in the RMT code base functions/methods related to de-registration of a system use `deregister`. A name change to `de_register`  makes it harder to find all the places in the code base where a de-registration happens. As such I'd urge you to reconsider this name change to help everyone else that may need to look for de-registration actions in the RMT code base.

Reply:
Resolved in 6fa9a9ea.